target/linux/realtek/files-5.10/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -3784,7 +4516,7 @@ static int rtl8218b_int_phy_probe(struct phy_device *phydev)
 		struct rtl83xx_shared_private *shared = phydev->shared->priv;
 		shared->name = "RTL8218B (internal)";
 		/* Configuration must be done while patching still possible */
-		return rtl8380_configure_int_rtl8218b(phydev);
+		ret = rtl8380_configure_int_rtl8218b(phydev);
```

Comment:
I get `error: 'ret' undeclared (first use in this function); did you mean 'net'?`